src/browser/provider/built-in/chrome/utils.js

```diff
@@ -0,0 +1,8 @@
+import fs from 'fs';
```

Comment:
There are a several promisified functions - https://github.com/DevExpress/testcafe/search?utf8=%E2%9C%93&q=promisify%28&type=
Let's combine it to the separate file - `src/utils/promisified-functions.js`